web/src/layout/main_old.jsx

```diff
@@ -163,6 +167,9 @@ const Main = ({ electricityMixMode, hasConnectionWarning }) => {
         <Tabs />
       </div>
       <OnboardingModal />
+      <InfoModal />
+      <FAQModal />
+      <SettingsModal />
```

Comment:
Any thoughts for a 'better' structure to incorporate any number of modals?

Reply:
We could do something more fancy, but don't think it's worth trying to make it too fancy and abstract 😄